--- better-cacti-templates-1.1.8-orig/scripts/ss_get_mysql_stats.php	2011-05-25 12:17:25.659963610 +0200
+++ better-cacti-templates-1.1.8/scripts/ss_get_mysql_stats.php	2011-05-25 12:19:29.509003208 +0200
@@ -68,6 +68,10 @@
 # ============================================================================
 # Include settings from an external config file (issue 39).
 # ============================================================================
+# prefer system location first
+if ( file_exists('/etc/webapps/cacti/ss_get_mysql_stats.php') ) {
+   require('/etc/webapps/cacti/ss_get_mysql_stats.php');
+}
 if ( file_exists(__FILE__ . '.cnf' ) ) {
    require(__FILE__ . '.cnf');
 }
--- better-cacti-templates-1.1.8/scripts/ss_get_by_ssh.php~	2011-12-21 18:26:28.000000000 +0200
+++ better-cacti-templates-1.1.8/scripts/ss_get_by_ssh.php	2011-12-21 18:29:47.234354418 +0200
@@ -66,6 +66,10 @@
 # ============================================================================
 # Include settings from an external config file (issue 39).
 # ============================================================================
+# prefer system location first
+if ( file_exists('/etc/webapps/cacti/ss_get_by_ssh.php') ) {
+   require('/etc/webapps/cacti/ss_get_by_ssh.php');
+}
 if ( file_exists(__FILE__ . '.cnf' ) ) {
    require(__FILE__ . '.cnf');
 }
